tardis/apps/publication_workflow/static/my-publications/my_publications.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * My Publications controller
+ *
+ */
+angular
+  .module('publicationWorkflow')
+  .controller('MyPublicationsController', function ($log, $resource, $timeout) {
+    const vm = this;
+
+    const draftPubsListRes = $resource('/apps/publication-workflow/draft_pubs_list/', {}, {
+      get: { method: 'GET', isArray: true },
+    });
+    const scheduledPubsListRes = $resource('/apps/publication-workflow/scheduled_pubs_list/', {}, {
+      get: { method: 'GET', isArray: true },
+    });
+    const releasedPubsListRes = $resource('/apps/publication-workflow/released_pubs_list/', {}, {
+      get: { method: 'GET', isArray: true },
+    });
+    const retractedPubsListRes = $resource('/apps/publication-workflow/retracted_pubs_list/', {}, {
+      get: { method: 'GET', isArray: true },
+    });
+
+    /**
+     * loadDraftPubsData
+     *
+     * Fetch the list of draft pubs available to the user
+     */
+    vm.loadDraftPubsData = function () {
+      draftPubsListRes.get().$promise.then((data) => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
            draftPubsListRes.get().$promise.then((data) => {
```